lib/nostrum/consumer.ex

```diff
@@ -247,17 +246,15 @@ defmodule Nostrum.Consumer do
     ConsumerSupervisor.start_link(
       __MODULE__,
       [mod, Keyword.drop(options, [:name])],
-      name: name,
-      spawn_opt: [Util.fullsweep_after()]
+      name: name
     )
   end
 
   def start_link(mod, options),
     do:
       ConsumerSupervisor.start_link(
         __MODULE__,
-        [mod, options],
-        spawn_opt: [Util.fullsweep_after()]
```

Comment:
i will get to these when i have linux access again

Reply:
I was able to add back this opt to the shard cache, but i cant here because ConsumerSupervisor.start_link doesnt accept most of the genserver options: https://hexdocs.pm/gen_stage/ConsumerSupervisor.html#t:option/0

Reply:
That makes sense, thanks!